shell/browser/web_dialog_helper.cc

```diff
@@ -153,10 +153,12 @@ class FileSelectHelper : public base::RefCounted<FileSelectHelper>,
                                                 paths[0].DirName());
         }
       }
+      // We should only call this if we have not cancelled the dialog
+      if (ready_to_call_listener)
+        OnFilesSelected(std::move(file_info), lister_base_dir_);
+    } else {
```

Comment:
This indentation is funky 🤔

Reply:
resolved in dm - it just looks funny but is correct 😆

Reply:
two-birds-one-stone-dept: the indentation will look less funky if we use positive rather than negative logic in the surrounding if-else:

```cc
if (cancelled) {
  OnSelectionCancelled();
} else {
  ...
  if (ready_to_call_listener)
    OnFIlesSelected(std::move(file_info), lister_base_dir_);
}